script/publish-pact.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-console */
+
+const pact = require('@pact-foundation/pact-node');
+const path = require('path');
+
+const consumerVersion = `1.0.0-${process.env.BUILD_NUMBER ||
+  Math.floor(new Date() / 1000)}`;
```

Comment:
Pact Broker requires both the consumer and provider to be explicit about their versions when generating the pact or verifying it.

`BUILD_NUMBER` was initially intended to be the build number from Jenkins, but maybe the version could be a working branch or the commit hash at HEAD.